packages/backend/src/open_payments/payment_pointer/service.test.ts

```diff
@@ -214,6 +215,49 @@ describe('Open Payments Payment Pointer Service', (): void => {
     })
   })
 
+  describe('Payment Pointer pagination', (): void => {
```

Comment:
Does this supersede the need for the other `getPageTests` call? In `assets` and `peers` we call  `getPageTests` (with different options) in both the resolver and the service.

I was expecting something like this here too:

```ts
    getPageTests({
      createModel: () => createPaymentPointer(deps),
      getPage: (pagination?: Pagination) =>
        paymentPointerService.getPage(pagination)
    })
```

Reply:
I really should have highlighted the line below. I'm referring the cases in the `test.each`.

Reply:
Oh, I missed the ones for the service. Thank you!

These are actually 2 different sets of tests, the ones for the resolvers (packages/backend/src/graphql/resolvers/page.test.ts) and the ones for the services (packages/backend/src/shared/baseModel.test.ts).

I'll add the service tests.